content/03.text-mining.md

```diff
@@ -7,8 +7,50 @@
 
 ### Constructing Databases and Manual Curation
 
-1. Talk about papers that construct knowledge graphs without text mining approaches
-2. Discuss the positives and negatives for these methods
+Constructing a database is an extensive process that can date back to the early 1990s [@pmid:7584356].
+This process involves gathering relevant text such as journal articles, abstracts, or web-based text.
+Each collection of text undergoes the process of tagging and normalizing important keywords to represent entities of interest.
+This task is commonly known as named entity recognition. 
+Typically, this task uses a dictionary look-up approach [@doi:10.1093/database/bas037] or a machine learning system [@doi:10.1093/bioinformatics/btw343; @doi:10.1142/9789812776136_0062; @doi:10.1186/1471-2105-12-S8-S5] to identify the keywords of interest.
+Following named entity recognition, the next step is to detect sentences that assert a relationship of interest (i.e. relationship extraction).
+This task can be accomplished via manual curation, automated approaches or a combination of both.
+Succeeding detection, identified sentences are incorporated as evidence for establishing a relationship between mentioned entities of interest. 
+
+Notable databases have been constructed using an extensive amount of manual curation (Table {@tbl:manual-curated-databases}).
+For example, COSMIC [@doi:10.1093/nar/gkw1121] was constructed via a group of domain experts scanning the literature for key cancer related genes.
+This database has reached close to 45k entries in 2016 and continues to grow throughout the years [@doi:10.1093/nar/gkw1121].
+Studies have shown that these databases contain relatively precise data, but in low quantities [@doi:10.1038/nmeth1209-860; @doi:10.1038/nmeth.1284; @doi:10.1093/database/bau058].
+This happens because the high publication rate is too much for curators to keep up [@doi:10.1007/s11192-010-0202-z].
+Plus, this process isn't completely error free as databases were found to be missing annotations and entries [@doi:10.1093/bioinformatics/btm229].
+These combined findings highlight a significant need for faster and more accurate approaches.
+
+There have been attempts to fix this bottleneck via semi-automatic ways [@doi:10.1016/j.jbi.2010.11.001; @doi:10.1093/database/bau067; @doi:10.1093/database/bas010; @doi:10.1093/database/baz068].
+One approach compared annotating PubMed queries with and without the help of an automated system [@doi:10.1016/j.jbi.2010.11.001].
+It was found that annotating with pre-annotated queries provided a better speed up than annotating from scratch.
```

Comment:
I think you're describing a process where a curator is supplied with a computer-annotated sentence (perhaps via some UI?). It took me a while to realize what this was about. It is probably good to assume your reader has less background and give them more context.